rpm_spec/subpackages/manageiq-ansible-venv

```diff
@@ -0,0 +1,14 @@
+%package ansible-venv
+Summary: %{product_summary} Ansible Runner Virtual Environment
+
+Requires: ansible
+Requires: ansible-runner
+Requires: python38
```

Comment:
I'm sure there are more requires here, but I'm not sure what - I'm assuming whatever I had -devel for I need the non-devel version.  This is part of what I want to verify at install time.

Reply:
I'm going to handle this in a follow up - I need to be able to exercise a playbook that uses the underlying packages directly before I'm confident anything else is needed here or not, but I don't want to hold up the progress.